TablePos/Properties/AssemblyInfo.cs

```diff
@@ -25,5 +25,5 @@
 //[assembly: AssemblyDelaySign(false)]
 //[assembly: AssemblyKeyFile("")]
 
-[assembly: AssemblyFileVersion("2.7.0")]
+[assembly: AssemblyFileVersion("2.7.2")]
```

Comment:
Why is this one 2.7.2 and the others 2.7.1?

Reply:
The fixes are are incremental of the sample application. Not across the whole samples